CHANGES.md

```diff
@@ -1,5 +1,36 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.4.0-M1
+
+### Breaking changes
+
+* The concept of atomic cancellation in channels is removed. All operations in channels
+    and corresponding `Flow` operators are cancellable in non-atomic way (#1813).
+* If `CoroutineDispatcher` throws `RejectedExecutionException`, schedule current `Job` cancellation in `Dispatchers.IO` (#2003). 
```

Comment:
Has to read. Proposed wording:
If `CoroutineDispatcher` throws `RejectedExecutionException`, cancel current `Job` and schedule its execution to `Dispatchers.IO` (#2003).